package.json

```diff
@@ -108,7 +111,8 @@
     "nock": "~12.0.3",
     "nyc": "~15.0.0",
     "standard-version": "^7.1.0",
-    "standardx": "^5.0.0"
+    "standardx": "^5.0.0",
+    "typescript": "^3.8.3"
```

Comment:
`typescript` should be a dev dependency, not a dependency.

Reply:
That's where it is already, in the `devDependencies` section?

Reply:
Oops. Looks right then.
The GitHub tool collapsed the file making it look like a dep.